src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The class to interact with users.
+ */
+public class Ui {
+    static final String LINE = "____________________________________________________________";
+
+    /**
+     * To welcome users.
+     */
+    public void hello() {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm T-Rex. Roarrrrrrrrrrrrrr!");
+        System.out.println("What do you need from me?");
+        System.out.println(LINE);
+        System.out.println();
+    }
+
+    /**
+     * To read user input.
+     *
+     * @return the user's input
+     */
+    public String readCMD() {
```

Comment:
```suggestion
    public String readCmd() {
```
Abbreviations and acronyms should not be uppercase when used as a part of a name